src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -701,8 +701,8 @@ function copyOptions() {
 
     // Number of recipient setup
     formatNumberBox($currRecipient.val(), NEW_QUESTION);
-    var $prevRadioButtons = $('table[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
-    var $currRadioButtons = $('table[class~="questionTable"]').last().find('input[name="numofrecipientstype"]');
+    var $prevRadioButtons = $('div[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
+    var $currRadioButtons = $('div[class~="questionTable"]').last().find('input[name="numofrecipientstype"]');
```

Comment:
Same here, I suggest that instead of relying on "divs" and classes, we select an ID (i.e. give an ID to the questionTable)